requirements.txt

```diff
@@ -12,19 +12,9 @@ anyconfig==0.12.0 \
     --hash=sha256:2119065ad835d0e4cbcb26a58fab0705952c5657c05fa90ef7a11cdecbbafb1f \
     --hash=sha256:3091d7675740686fade85755537f2a7c6ccefa8659c5bb56174e02ded74b96d5
     # via -r ../../requirements.in
-apache-libcloud==3.3.1 \
-    --hash=sha256:c3722c4dd58b0ee4beaf4e615e0ba82505d9a915e55319dc84383687166a3eed \
-    --hash=sha256:d7450453eaf5904eb4fb4f74cf9f37dc83721a719bce34f5abb336b1a1ab974d
-    # via -r ../../requirements.in
-<<<<<<< HEAD
```

Comment:
We should not have any commit with broken syntax. So, need to fix all these merge conflicts in the second commit of this PR.

Reply:
fixed